connectors/es/sink.py

```diff
@@ -684,14 +748,16 @@ async def enqueue_docs_to_delete(self, existing_ids):
                     "_id": doc_id,
                 }
             )
-            self.total_docs_deleted += 1
+            self.counters.increment(DELETES_QUEUED)
 
-    def _log_progress(self):
+    def _log_progress(
+        self,
+    ):
         self._logger.info(
             "Sync progress -- "
-            f"created: {self.total_docs_created} | "
-            f"updated: {self.total_docs_updated} | "
-            f"deleted: {self.total_docs_deleted}"
+            f"created: {self.counters.get(CREATES_QUEUED)} | "
+            f"updated: {self.counters.get(UPDATES_QUEUED)} | "
+            f"deleted: {self.counters.get(DELETES_QUEUED)}"
```

Comment:
So now it also counts queued operations, not executed, right?

Reply:
It was always counting queued operations. Now it also counts executions in the final result. ;) 

Do you want me to look at changing the progress logs to be based on executed results, rather than queued operations?

Reply:
> Now it also counts executions in the final result. ;)
I love it!

> Do you want me to look at changing the progress logs to be based on executed results, rather than queued operations?

Not in this PR definitely, but I think it might be worth to take a look at it - it's great that now with this change we have options!